addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -148,7 +148,7 @@ var FileWidget = SearchableMediaWidget.extend({
     }),
     existingAttachmentsTemplate: undefined,
 
-    IMAGE_MIMETYPES: ['image/jpg', 'image/jpeg', 'image/jpe', 'image/png', 'image/svg+xml', 'image/gif'],
+    IMAGE_MIMETYPES: ['image/jpg', 'image/jpeg', 'image/jpe', 'image/png', 'image/svg+xml', 'image/gif', 'image/webp'],
```

Comment:
Do we want to support those ? Not saying we shouldn't, but I did not know that this was part of the spec ? cc @SimonGenin

Reply:
we are doing it when pasting content also so why not while checking for images. https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L4708 cc @SimonGenin

Reply:
@shsa-odoo @dmo-odoo I think we should yes, it a recommended format for the web

Reply:
@shsa-odoo  & @dmo-odoo ,
We should support webp, at least from version 16.4.
See work done here:
https://github.com/odoo/odoo/pull/85494